packages/platform-android/native_modules.gradle

```diff
@@ -126,11 +126,15 @@ class ReactNativeModules {
     String packageClassInstances = ""
 
     if (packages.size() > 0) {
-      packageImports = "import ${packageName}.BuildConfig;\nimport ${packageName}.R;\n\n"
+      def interpolateDynamicValues = {
+        it.replace("{{ packageName }}", packageName)
+      }
       packageImports = packageImports + packages.collect {
```

Comment:
Sure! I left the `+=` specifically in case someone would add something to that variable before the `if` or as the initial value — this would be more failsafe, but I guess we can remove this safely.
```suggestion
      packageImports = packages.collect {
```

Reply:
Huh, interesting thing is happening with that suggestion, it removes the initial space too…
```suggestion
       packageImports = packages.collect {
```